src/t2iapi/combined/combined_requests.proto

```diff
@@ -70,3 +70,14 @@ message SetModeOfOperationAndSetOperatingModeRequest {
                                             // shall be set
     t2iapi.operation.OperatingMode operating_mode = 4; // OperatingMode value to be set for the operation
 }
+
+/*
+Request to set the @ActivationState of the system context with the specified handle to the requested value and to set
+the @ContextAssociation of a context state from the context descriptor with the given handle to the requested value.
+*/
+message SetSystemContextActivationStateAndContextAssociationRequest {
+    string system_context_handle = 1; // handle of the system context for which the @ActivationState shall be set
+    t2iapi.activation_state.ComponentActivation system_context_activation = 2; // the @ActivationState the system context shall be set to
```

Comment:
it is the requested ComponentActivation value

Reply:
changed the comment